parameter/XmlFileIncluderElement.cpp

```diff
@@ -120,9 +119,9 @@ std::string CXmlFileIncluderElement::getIncludedElementType() const
 {
     std::string strKind = getKind();
 
-    int iPosToRemoveFrom = strKind.rfind("Include", -1);
+    std::string::size_type uiPosToRemoveFrom = strKind.rfind("Include", std::string::npos);
```

Comment:
There are some inconsistencies: in most places, the "ui" prefix has been kept but where there previously was a "i" prefix (i.e. in RuleParser), the "i" prefix was removed.